src/ansys/aedt/core/q3d.py

```diff
@@ -1400,14 +1400,27 @@ def net_sources(self, net_name):
         """
         sources = []
         net_id = -1
```

Comment:
```suggestion
        net_sources_sinks = next(
    (i.children for i in self.boundaries_by_type.get("SignalNet", []) if i.name == net_name), {}
)
net_id = next(
    (i.props.get("ID", -1) for i in self.boundaries_by_type.get("SignalNet", []) if i.name == net_name), -1
)
```
try it on your side first if you like.
I can't select all the lines but it's from net_id = -1 till the break inside the for loop: for i in signal_nets.